app/styles/components/c-contact-form.scss

```diff
@@ -0,0 +1,121 @@
+@import '../settings';
+
+.c-contact-form {
+  margin: 0 auto;
+  max-width: 480px;
+  width: 100%;
+
+  h1 {
+    color: $color-4;
+    font-size: $font-size-xxx-big;
+    font-weight: $font-weight-bold;
+    letter-spacing: -1px;
+    margin: 40px 0 30px;
+    text-align: center;
+  }
+
+  p {
+    color: $color-4;
+    font-size: $font-size-x-medium;
+    margin-bottom: 30px;
+    text-align: center;
+  }
+
+  label,
+  input,
+  textarea,
+  select {
+    color: $color-9;
+    display: block;
+    width: 100%;
+  }
+
+  label {
+    color: $color-4;
+    font-size: $font-size-small;
+    margin-bottom: 5px;
+    text-transform: uppercase;
+  }
+
+  input {
+    background-color: $color-4;
+    border: solid 1px $color-10;
+    margin-bottom: 10px;
+    outline: 0;
+    padding: 10px;
+  }
+
+  .select-container {
+    position: relative;
+
+    &::after {
+      background: transparent;
+      border-bottom: $color-9 2px solid;
+      border-left: $color-9 2px solid;
+      content: '';
+      height: 10px;
+      position: absolute;
+      right: 15px;
+      top: 12px;
+      transform: rotate(-45deg);
+      width: 10px;
+    }
+  }
+
+  select {
+    -moz-appearance: none;
+    -webkit-appearance:none;
+    background-color: $color-4;
+    border: solid 1px $color-10;
+    border-radius: 0;
+    color: $color-5;
+    cursor: pointer;
+    font-size: $font-size-small;
+    height: 40px;
+    margin-bottom: 10px;
+    outline: 0;
+    padding: 10px;
+
+    &::-ms-expand {
+      display: none;
+    }
+  }
+
+  textarea {
+    background-color: $color-4;
+    border: solid 1px $color-10;
+    border-radius: 0;
+    height: 200px;
+    margin-bottom: 10px;
+    outline: 0;
+    padding: 10px;
+  }
+
+  .submit-contact {
```

Comment:
We have this button styles in other parts of the site: that's clearly a component. Use `c-button` component to set all components you find similar and set as many modifiers (`-red`, `-big`, `-whatever`) you need.

This way, you don't have to repeat button styles every time you find a button.